chroma_core/models/filesystem.py

```diff
@@ -109,6 +109,20 @@ def get_servers(self):
         # (http://stackoverflow.com/questions/4764110/django-template-cant-loop-defaultdict)
         return dict(servers)
 
+    def get_server_groups(self):
```

Comment:
Where does this get called?

Reply:
from my task-hotpools patch, I could include it there instead.